sw/device/lib/testing/test_rom/BUILD

```diff
@@ -92,7 +92,7 @@ cc_library(
         "//sw/device/lib/base:abs_mmio",
         "//sw/device/lib/base:macros",
         "//sw/device/lib/base:multibits",
-        "//sw/device/silicon_creator/lib:epmp",
+        "//sw/device/silicon_creator/lib:epmp_state",
```

Comment:
Thanks for this rename. IIRC this library is only responsible for the in-memory copy of the ePMP configuration, right?

Reply:
Yes